feg/gateway/services/session_proxy/servicers/session_controller_test.go

```diff
@@ -1295,6 +1355,7 @@ func initMconfig() error {
                }
             },
             "gy": {
+			   "disableGy": false,
```

Comment:
fmt?

Reply:
Do you mean format? I don't see any issue there.

![image](https://user-images.githubusercontent.com/16157139/89888270-fac2b900-dbcf-11ea-9dec-63f8bd42cf7e.png)

Reply:
Maybe just the github view ... 🤦‍♂️